docs/_asset/index.css

```diff
@@ -762,10 +757,21 @@ button.success {
   mask-image: paint(squircle);
   --squircle-radius: 10px;
   border-radius: 10px;
+
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+
+  min-height: 10rem;
+  height: 24rem;
```

Comment:
these 2 can now be more easily adjusted to define the initial height of the panels